docs/api/Auth0.AuthenticationApi.Models.PasswordlessEmailRequest.html

```diff
@@ -147,6 +147,31 @@ <h5 class="decalaration">Declaration</h5>
   <div class="codewrapper">
     <pre><code class="lang-csharp hljs">[JsonProperty(&quot;client_id&quot;)]
 public string ClientId { get; set; }</code></pre>
+  </div>
+  <h5 class="propertyValue">Property Value</h5>
+  <table class="table table-bordered table-striped table-condensed">
+    <thead>
+      <tr>
+        <th>Type</th>
+        <th>Description</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td><span class="xref">System.String</span></td>
+        <td></td>
+      </tr>
+    </tbody>
+  </table>
+  <a id="Auth0_AuthenticationApi_Models_PasswordlessEmailRequest_ClientSecret_" data-uid="Auth0.AuthenticationApi.Models.PasswordlessEmailRequest.ClientSecret*"></a>
+  <h4 id="Auth0_AuthenticationApi_Models_PasswordlessEmailRequest_ClientSecret" data-uid="Auth0.AuthenticationApi.Models.PasswordlessEmailRequest.ClientSecret">ClientSecret</h4>
+  <div class="markdown level1 summary"><p>Client Secret of the application.</p>
+</div>
+  <div class="markdown level1 conceptual"></div>
+  <h5 class="decalaration">Declaration</h5>
+  <div class="codewrapper">
+    <pre><code class="lang-csharp hljs">[JsonProperty(&quot;client_secret&quot;)]
+public string ClientSecret { get; set; }</code></pre>
```

Comment:
Looks like docs wasnt run when adding this property 5 months ago https://github.com/auth0/auth0.net/commit/b7d2d8e0b6034aed46f07f83b06adc62553fd4a7